### DISCUSSION TITLE
Asterisk: Send Action Fully Functional?

### DISCUSSION THREAD

**Discussion by msettlemeyer** [TECH7Fox/Asterisk-integration#96] [General]
Hi @TECH7Fox , just wondering if this service is fully functional. Tried running a command with debug logging on, but I'm not seeing the results in the log.

Command in question is CoreShowChannels
Parameters are
ActionId: 11111 (just to have a value here)

**Comment by TECH7Fox** [comment]
Currently the call action service does not return anything. When I made this that feature didn't exist yet.

I will add support for this.

But the service call itself does work, and you will see the action being made in Asterisk, but you can't get the result in HA. Yet.
